go/vt/vtorc/logic/topology_recovery.go

```diff
@@ -882,3 +893,38 @@ func fixReplica(ctx context.Context, analysisEntry *inst.ReplicationAnalysis) (r
 	err = setReplicationSource(ctx, analyzedTablet, primaryTablet, reparentutil.IsReplicaSemiSync(durabilityPolicy, primaryTablet, analyzedTablet))
 	return true, topologyRecovery, err
 }
+
+// recoverErrantGTIDDetected changes the tablet type of a replica tablet that has errant GTIDs.
+func recoverErrantGTIDDetected(ctx context.Context, analysisEntry *inst.ReplicationAnalysis) (recoveryAttempted bool, topologyRecovery *TopologyRecovery, err error) {
```

Comment:
Here (in this function) is where we want to check `config.ConvertTabletWithErrantGTIDs()`

Reply:
Instead done in `getCheckAndRecoverFunctionCode`